docs/en/deploy/Running-Alluxio-On-Docker.md

```diff
@@ -167,7 +170,7 @@ $ docker run -d --rm \
     --name=alluxio-worker \
     --shm-size=1G \
     -v /alluxio_ufs:/opt/alluxio/underFSStorage \
-    -e ALLUXIO_JAVA_OPTS="-Dalluxio.worker.memory.size=1G  -Dalluxio.master.hostname=$(hostname -i)  -Dalluxio.worker.hostname=$(hostname -i)" \
+    -e ALLUXIO_JAVA_OPTS="-Dalluxio.worker.memory.size=1G  -Dalluxio.master.hostname=alluxio-master  -Dalluxio.worker.hostname=alluxio-worker" \
```

Comment:
`$(hostname -i)` will give the host IP, which is incorrect. Containers can be found by name.

Reply:
is this true?